### ISSUE TITLE
Support the Leadville Trail 100

### ISSUE THREAD

**Issue by ctm** [ctm/fantoccini-shiprock#2]
LT100 is Athlinks, so in theory it should be trivial to support.

**Comment by ctm** [comment]
The problem with the Leadville 100, is that even after specifying the race, it's necessary to request the "Full Course" splits. I should just be able look for that as an option on the landing page and choose it if it's not already chosen.  Beyond that, I think athlinks reports a few (there were 3 in 2019, I believe) finishers that finish over 30 hours and those aren't reported to ultrasignup, but I think it's fine to leave them in, especially since I'm not doing anything with these results right now.

**Comment by ctm** [comment]
It's a bit trickier than I thought. The athlinks driver doesn't currently interact with their pop-ups, instead we rely on a "View All" link showing up on the landing page and that's how we select which race to go to. It's only after we get to the race page where we need to select "Full Course", and to do that requires interacting with their pop-ups.

Currently, athlinks only implements Run For The Zoo (5k, 10k and half marathon) and Leadville 100 for 2019, so a quick workaround would simply to start at the results page with the splits set to Full course, so we'd just have to hard-code in four different urls (one for each of the three RFTZ races and one for LT100). That doesn't scale for more years.

So, I'll implement that quick hack, but also create a new issue that involves exploring how to drive their pop-ups. FWIW, I dont think that is particularly hard, but considering I have much more important things to do, than to play with fantoccini-shiprock, I don't think it's something I should do today.

**Comment by ctm** [comment]
FWIW, I added support for driving the pop-ups and made it so we get a lot of years, but we don't get some early years due to them not having the sex and age of the participants.

**Comment by ctm** [comment]
Since we didn't get all the years, I should have either reopened this issue or created a new issue. I meant to create a new issue, but I forgot to.
